### DISCUSSION TITLE
Where are the outputs stored for StableStudio?

### DISCUSSION THREAD

**Discussion by metaphorz** [Stability-AI/StableStudio#32] [General]
I could not locate where the output/generated images are stored on the MacBook where I am running it (localhost:3000). For stable-diffusion-webui, there is an output folder where those images are located.

**Comment by KAJdev** [comment]
if you are running StableStudio via stable-diffusion-webui, then the outputs would be saved according to stable-diffusion-webui, not StableStudio.

**Comment by metaphorz** [comment]
I am running StableStudio separately. I just pointed out stable-diffusion-webui as an example software where an 'outputs folder' is used. In playing around with StableStudio some more, I see that there is a convenient download images icon so that is good and will work.

**Comment by cruhl** [comment]
For reference, when using the default Stability API plugin, images are stored in the cloud.
